docs/libertyDev.md

```diff
@@ -101,12 +101,24 @@ The following is a technology preview. The features and parameters described bel
 
 Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `libertyDevc` task is to specify the `libertyDev` task with the `--container` option. 
 
-The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described below.
-
 You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. You can examine the commands used to build and run the container by viewing the console output of dev mode.
 
 When dev mode runs with container support it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. build/wlp/usr/servers/defaultServer/logs.
 
+The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described.
+
+Dev mode offers different levels of file tracking and deployment depending on the way the file is specified in the Dockerfile. 
+1. When you use the COPY command on an individual file dev mode can track file changes and hot deploy them to the container subject to the limitations below. **This is the most productive way to deploy files.**
+- E.g. `COPY src/main/liberty/config/server.xml /config/` 
+- Note that the Dockerfile must copy only one .war file for the application.  Other application archive formats or multiple .war files are not supported.
```

Comment:
I don't think we need to say "Other application archive formats or multiple .war files are not supported.".  They are allowed but dev mode will not delete those lines from the temporary Dockerfile.  So if the files are not present at docker build time, the docker build will fail.  But there's nothing particularly wrong with having other formats.

Reply:
Ok, so just multiple war files will lead to problems (they are removed).